src/scripts/main.js

```diff
@@ -1 +1,38 @@
 'use strict';
+
+const navBtn = document.querySelector('#nav-menu-trigger');
+const nav = document.querySelector('.nav');
+const navLogo = document.querySelector('.banner__logo');
+
+navBtn.addEventListener('click', () => {
+  nav.classList.toggle('nav--show');
+  navBtn.classList.toggle('banner__menu-btn--close');
+  navLogo.classList.toggle('banner__logo--black');
+});
+
+const navLink = document.querySelectorAll('.nav__link--scroll');
+
+navLink.forEach(link => {
```

Comment:
You can use event delegation here.
Basically, it means, that you could add a listener for the list **only**, and don't add listeners for each nav lin.k

More info here - https://javascript.info/event-delegation